### PR TITLE
Added epel repositories for Linux on Z(s390x)

### PR DESCRIPTION
**Pull request by ketank-ibm**
Repositories were required to download specific dependencies. To support the same we have added necessary repositories for Linux on Z(s390x)
Also added support for Linux on Z(s390x)

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @salelkar-ibm

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: attributes/epel.rb [hunk 1]
```diff
@@ -1,24 +1,31 @@
 default['yum']['epel']['repositoryid'] = 'epel'
 
-case node['platform']
-when 'amazon'
-  default['yum']['epel']['description'] = 'Extra Packages for Enterprise Linux 6 - $basearch'
-  default['yum']['epel']['mirrorlist'] = 'http://mirrors.fedoraproject.org/mirrorlist?repo=epel-6&arch=$basearch'
-  default['yum']['epel']['gpgkey'] = 'http://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-6'
+case node[:kernel][:machine]
```
Comment (tas50) [inline attributes/epel.rb]: This should be a string not a symbol so we don't mix attribute access methods

Comment (ketank-ibm) [inline attributes/epel.rb]: Changes the values to string.

**Comment by tas50** [comment]
@ketank-ibm Does zlinux report as zlinux with node['platform']?

**Comment by cwsolee** [comment]
Hi, We(IBM) have been working with chef and verified a number of cookbooks on linux on z but supermarket right now don't quite provide an indicator for user to know which cookbook is verified on LoZ. Adding this flag is what was suggested by Chef team. Hence, we're do this same change on all cookbooks that has been verified.

**Comment by ketank-new** [comment]
Hi ,
I would like to know,if there is Continuous Integration setup for Cook Books
If yes than how can we make s390x platform part of the CI process

Regards
Ketan

**Pull request merged**